app.py

```diff
@@ -521,12 +521,14 @@ def get_quiz(level_source, question_nr, attempt):
     question = quiz.get_question(quiz_data, question_nr)
     question_obj = quiz.question_options_for(question)
 
+    chosen_letter = 'X' if session.get('chosenOption') is None else session.get('chosenOption')
```

Comment:
Waarom is dit een 'X' en geen None? Zou dat niet duidelijker zijn?

Reply:
Bij deze expressie: if question.correct_answer in chosen_option, dan krijg ik een error omdat chosen_optiion soms None kan zijn. Dus daarom heb ik chosen_option een random letter gegeven als het None is. Dan kan je wel het correcte antwoord vergelijken met de letter.

Reply:
Ik zou overigens X wel de string 'None' kunnen geven. Is dit een betere oplossing?
Enige probleem is dan wel de letter E in None. Mochten er 5 opties zijn. Daarom heb ik X gekozen als letter, die zit aan het einde van het alfabet

Reply:
Ik heb het toch liever met None want dat is netter. 

Dus gewoon: chosen_letter = session.get('chosenOption')

Je kunt dan om de rest van de code een `if not chosen_letter is None` heen zetten zodat het niet mis gaat als er niks gekozen is. Het is mischien ook de moeite waard om te onderzoeken waarom het None kan zijn. Verwachten we dat? In welke situatie?

Reply:
We verwachten None in de situatie dat je begint met een vraag, en er nog geen letter is gekozen
dus je bedoeld qua code?

`if not chosen_option is None
  if question,correct in chosen_option
`

Reply:
Inmiddels is het al opgelost Lotus dus jij hoeft niets meer te doen. Kijk maar even naar de veranderingen in het PR om ervan te leren